single_cell_analysis/scripts/util/util.R

```diff
@@ -56,16 +56,16 @@ kmeans_eval <- function(feature, celltype, iter = 50, seed = 1234){
                     nmi_var = round(sd(nmi_score_all), 2), ari_var = round(sd(ari_score_all), 2)))
 }
 
-cal_performance <- function(pred, class, option){
+cal_performance <- function(pred, type, option){
```

Comment:
these variable names also seem a bit off - I think something more like `pred_class` and `true_class` would be easier to track

Reply:
option is also not specific enough. something like `num_class_labels` is better